assets/javascripts/kitten/components/box/link-box.js

```diff
@@ -135,23 +134,25 @@ const styles = {
   paragraph: {
     display: 'flex',
     flexDirection: 'column',
-    flexGrow: 1,
     justifyContent: 'center',
-    padding: '10px 18px 15px 20px',
+    flexGrow: 1,
+    paddingTop: 10,
+    paddingLeft: 18,
```

Comment:
C'est pas sensé être 
```suggestion
    paddingLeft: 20,
```
si on veut respecter `padding: '10px 18px 15px 20px'` ?
L'ordre est ⬆️ ➡️ ⬇️ ⬅️.